app/stylesheets/boilerplate/header.scss

```diff
@@ -1,10 +1,86 @@
+@import '../config/dimensions';
+@import '../config/type';
+@import '../config/colours';
+@import '../config/list';
+
 .header {
+  padding-left: $header-common-margin;
+  padding-right: $header-common-margin;
 
   .logo {
     float: left;
+
+    img {
+      float: left;
+      margin-right: $header-common-margin;
+      margin-top: $logo-top-position;
+      width: $header-height;
+    }
+  }
+
+  h1 {
+    color: $theme-blue-alt;
+    display: inline-block;
+    float: left;
+    font-size: $dash-title;
+    font-weight: $title-weight;
+    line-height: $dash-title + 2;
+    margin-top: $header-common-margin;
+    text-transform: uppercase;
+    width: $tab-label-width;
   }
 
+  // main nav
+
   .header-nav {
     float: right;
+    margin-top: $nav-top-position;
+
+    & > ul {
```

Comment:
careful, these styles should go in stylesheets for the child components, not here. Only rules for html in the component should go in its styles.